### ISSUE TITLE
Game crashes after few minutes during calls.

### ISSUE THREAD

**Issue by MrRetr0** [bowser0000/SkyblockMod#128]
This is a weird one. Essentially, if I'm in a discord call while also playing Skyblock with the mod installed, my game crashes after a few minutes of playtime. No errors are given, it is simply a desktop crash. I've attached the latest crash log.
[crash-2022-10-01_16.02.56-client.txt](https://github.com/bowser0000/SkyblockMod/files/9691175/crash-2022-10-01_16.02.56-client.txt)

**Comment by hannibal002** [comment]
```
java.lang.StringIndexOutOfBoundsException: String index out of range: 1
	at java.lang.String.substring(String.java:1951)
	at me.Danker.utils.Utils.capitalizeString(Utils.java:197)
	at me.Danker.features.GemstonesLore.onTooltip(GemstonesLore.java:59)
```
@MrRetr0 Does this error occur every time you have a specific item in your inventory? Like a drill or weapon with gemstones applied to it? If so, what item exactly?

**Comment by cinqdtun** [comment]
The problem has been fixed in development branche you just need to download source code from https://github.com/bowser0000/SkyblockMod/tree/master and compile it with gradlew and fix your crash !

**Comment by hannibal002** [comment]
Is there a reason why the action tab is not visible? Helping the mod needs more effort :/

**Comment by cinqdtun** [comment]
What do you mean by "action tab" ?  @hannibal002

**Comment by hannibal002** [comment]
> What do you mean by "action tab" ? @hannibal002

![image](https://user-images.githubusercontent.com/24389977/193448513-608fa7be-0777-4430-b5c6-a57e352278dd.png)
![image](https://user-images.githubusercontent.com/24389977/193448520-2eb34630-8d35-4679-98bd-1acb3a5e2b85.png)

**Comment by cinqdtun** [comment]
Okay i see yes you can also get mod updated hre

**Comment by hannibal002** [comment]
Normally, yes. But since DSM has the tab disabled (for non members or globally?) I need to compile the mod myself instead of just download the precompiled jar from the action tab like for other mods.

**Comment by bowser0000** [comment]
> Normally, yes. But since DSM has the tab disabled (for non members or globally?) I need to compile the mod myself instead of just download the precompiled jar from the action tab like for other mods.

The actions tab should be visible for everyone, have you tried seeing if it's visible on incognito or a new browser.

**Comment by bowser0000** [comment]
Fixed in 1.9
